### ISSUE TITLE
Horizontal scroll on mobile

### ISSUE THREAD

**Issue by tofanelli** [alvarotrigo/multiscroll.js#71]
Hey Alvaro! How's going? hope ok...

Look, i was wondering if its possible to change the orientation from vertical to horizontal on mobile views (max-width: 700px).... i think would have an interesting result instead narrowed sections. And, in this case, instead the sections go up and down, they would go right and left to "disappear".

This would be a nice addition? 

Cheers =D

**Comment by alvarotrigo** [comment]
> This would be a nice addition?

Definitely. Feel free to add a PR.

**Comment by tofanelli** [comment]
Superb! Glad you liked!

I'm not good at JS as you, but I think what should be added would be something like

e(".ms-right, .ms-left").css({width:"100%",position:"absolute",height:"50%","-ms-touch-action":"none"}

but for mobile views....  is it right?

**Comment by alvarotrigo** [comment]
You'll need a lot more code than just that :)

**Comment by tofanelli** [comment]
Yeah.... i just noticed this...let's pull the sleeves and get this job done =D

**Comment by tofanelli** [comment]
Hey alvaro... how's going?

I know I shouldn't do this, but.... do you have any plans to take a look into this? I was planning to make a website with this layout =)


cheers =)

**Comment by alvarotrigo** [comment]
@tofanelli nop. No time for it.

**Comment by tofanelli** [comment]
One year later.... and still waiting =)

**Comment by alvarotrigo** [comment]
@tofanelli not planning to work on this feature in the short time. 
I decided to adopt a more responsive solution by releasing the [Responsive Expand extension](https://alvarotrigo.com/multiScroll/extensions/responsive-expand.html).

It seems to be a better solution from the user experience point of view.

**Comment by tofanelli** [comment]
Yeaahhhh... but now it seems just an ordinary website made with a few css or using any regular framework.... Don't spark in my eyes.... At least for me... =\

**Comment by alvarotrigo** [comment]
Nothing better than a real responsive site for small screen devices with slow rendering engines.

**Comment by tofanelli** [comment]
Yeah... kinda of... missing some UX sparkle... something to really shine on... 

But I agree with you, slow rendering engines kills any gorgeous job implemented, but, its not the average =)
